### PR TITLE
Update create_admin_account.rst

### LINKED ISSUE #3015: docs: italicize Journalist Interface
# Bug

## Description

The [documentation style guidelines](https://docs.securedrop.org/en/latest/development/documentation_guidelines.html#style-guide) reads "When SecureDrop-specific terminology is used, italicize it.".

Look for the string **Journalist Interface** in [the documentation](https://github.com/freedomofpress/securedrop/blob/develop/docs/) and make sure all occurrences are italicized. For instance [create_admin_account.rst](https://github.com/freedomofpress/securedrop/blob/develop/docs/create_admin_account.rst) is missing some.

### PR DESCRIPTION
**Pull request by vaibssingh**
## Status

Ready for review

## Description of Changes

Fixes #3015 

Italicized *Journalist Interface*'s

### If you made changes to documentation:

- [x] Doc linting passed locally

### CODE REVIEW

File: docs/create_admin_account.rst [hunk 1]
```diff
@@ -1,30 +1,30 @@
-Create an admin account on the Journalist Interface
+Create an admin account on the *Journalist Interface*
 ===================================================
```
Comment (redshiftzero) [inline docs/create_admin_account.rst]: Linting is failing due to the title not being long enough: 

```
/src/docs/create_admin_account.rst:2: (WARNING/2) Title underline too short.
```

You can run the linting locally via `make docs-lint` following [the documentation here](https://docs.securedrop.org/en/latest/development/documentation_guidelines.html#testing-documentation-changes).

**Comment by vaibssingh** [comment]
@redshiftzero Fixed!

**Comment by redshiftzero** [comment]
Nice job @vaibssingh - all checks are passing now. One last thing prior to merge: could you [squash these two commits](https://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history) together?

**Comment by vaibssingh** [comment]
Please do! I have been away for a while and it will take me some more time
to get back. And sorry for the delay so far :|

**Comment by dachary** [comment]
@redshiftzero squashed, rebased and repushed with @vaibssingh permission :-)

**Comment by vaibssingh** [comment]
Thanks a lot and sorry again. Next time I will make sure of my availability
before starting a PR.

**Comment by dachary** [comment]
@vaibssingh this is fine really, thanks for your contribution :-)

**Pull request merged**